repo/packages/Q/quobyte/0/config.json

```diff
@@ -0,0 +1,110 @@
+{
+  "type": "object",
+  "properties": {
+    "mesos": {
+      "depscription": "Mesos specific configuration properties",
+      "properties": {
+        "master": {
+          "default": "zk://master.mesos:2181/mesos",
+          "description": "The URL of the Mesos master. The format is a comma-delimited list of hosts like zk://host1:port,host2:port/mesos. If using ZooKeeper, pay particular attention to the leading zk:// and trailing /mesos! If not using ZooKeeper, standard host:port patterns, like localhost:5050 or 10.0.0.5:5050,10.0.0.6:5050, are also acceptable.",
+          "type": "string"
+        }
+      },
+      "required": ["master"],
+      "type": "object"
+    },
+    "quobyte": {
+      "type": "object",
+      "description": "Quobyte Framework specific configuration properties",
+      "properties": {
+        "cpus": {
+          "default": 1.0,
+          "description": "CPU shares to allocate to each Quobyte scheduler instance.",
+          "minimum": 0.1,
+          "type": "number"
+        },
+        "mem": {
+          "default": 1024.0,
+          "description": "Memory (MB) to allocate to each Quboyte scheduler instance.",
+          "minimum": 768.0,
+          "type": "number"
+        },
+        "instances": {
+          "default": 1,
+          "description": "Number of Quobyte framework instances to run.",
+          "minimum": 1,
+          "type": "integer"
+        },
+        "framework-name": {
+          "type": "string",
+          "description": "The name of the framework.",
+          "default": "quobyte"
+        },
+        "zk": {
+          "type": "string",
+          "description": "Name or IP address of the zookeeper server.",
+          "default": "master.mesos:2181"
+          },
+        "port": {
+          "type": "string",
+          "description": "Port to which the framework's HTTP server is bound.",
+          "default": "7878"
+          },
+        "docker_image": {
```

Comment:
This is a different docker image used in tasks scheduled by the framework but nevertheless i added it to resources.